setup.py

```diff
@@ -301,7 +301,6 @@ def make_release_tree(self, base_dir, files):
 
     package_dir={"": "lib"},
     packages=find_packages("lib"),
```

Comment:
Does this need to be changed to `find_namespace_packages`? 

The [docs for setuptools](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-namespace-packages) indicate that packages without `__init__` will be omitted, and as mpl_toolkits no longer has a `__init__` I suspect it falls into that category